src/sentry/migrations/0020_auto_20191125_1420.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2019-11-25 14:20
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from sentry import eventstore
+from sentry.models import EventAttachment
+from sentry.utils.query import RangeQuerySetWrapper
+
+
+def forwards(apps, schema_editor):
```

Comment:
Could you figure out a way to make this importable, so we can easily run this offline?

Reply:
Does the new function work for you or do you need it moved somewhere outside the migration?

Reply:
```python
from sentry import EventAttachment

sentry = __import__("sentry.migrations.0020_auto_20191125_1420")
backfill = getattr(sentry.migrations, '0020_auto_20191125_1420').backfill_group_ids

backfill(EventAttachment)
```